clicommand/env_get.go

```diff
@@ -0,0 +1,143 @@
+package clicommand
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/buildkite/agent/v3/env"
+	"github.com/buildkite/agent/v3/jobapi"
+	"github.com/urfave/cli"
+)
+
+const envClientErrMessage = `Could not create Job API client: %v
+This command can only be used from hooks or plugins running under a job executor
+where the "job-api" experiment is enabled.
+`
+
+const envGetHelpDescription = `Usage:
+  buildkite-agent env get [variables]
```

Comment:
IIRC there's some weirdness with the number of newlines after `Usage:`. A lot (all?) of the other commands have 2 newlines whereas there's just one here. Might be best to make a docs PR at the same time to check.

Reply:
I've made the new subcommands use 2, good idea for a follow-up.